### ISSUE TITLE
`from > to` results in bad interval

### ISSUE THREAD

**Issue by MatthewCaseres** [JuliaActuary/ExperienceAnalysis.jl#10]
```jl
exposure(Anniversary(Month(1)), Date(2013, 1, 31), Date(2012, 4, 1))
```
gives
```jl
1-element Vector{NamedTuple{(:from, :to), Tuple{Date, Date}}}:
 (from = Date("2013-01-31"), to = Date("2012-04-01"))
```

**Comment by MatthewCaseres** [comment]
I think this should just give an empty array, then when we flatten the rows will just disappear.

**Comment by alecloudenback** [comment]
Good catch - I think that this should error rather than silently disappear. I can't really think of a scneario where I'd willinging pass data here, have some end dates behind the start, but still want to return partial cases.

Something like `to ≤ from && DomainError("from=$from argument is a later date than the to=$to argument. Did you accidentally reverse the arguments?")` would suffice, you think?

**Comment by MatthewCaseres** [comment]
agree, throwing an error makes sense

**Comment by Abubakrce19** [comment]
Plz assign me

**Comment by MatthewCaseres** [comment]
this should have been closed when PR was merged
